server/src/main/java/org/apache/iotdb/db/utils/AuthUtils.java

```diff
@@ -188,10 +188,14 @@ public static boolean validatePassword(String originPassword, String encryptPass
    *
    * @param pathA sub-path
    * @param pathB path
-   * @return True if pathA == pathB, or pathA is an extension of pathB, e.g. pathA = "root.a.b.c"
-   *     and pathB = "root.a"
+   * @return True if pathA == pathB, or pathA is an extension of pathB, or pathA is contained in the
+   *     semantics of "**" in pathB, e.g. pathA = "root.a.b.c" and pathB = "root.a" or pathB =
+   *     "root.a.**"
    */
   public static boolean pathBelongsTo(String pathA, String pathB) {
+    if (pathB.endsWith("**")) {
```

Comment:
Can it handle the paths like `root.sg.d.*`?

Reply:
OK, I'll add this feature.